src/services.py

```diff
@@ -0,0 +1,121 @@
+from telegram import Bot, ChatMember, Message, Update
+from telegram import User as TelegramUser
```

Comment:
```suggestion
from telegram import User as TelegramBotUser
```
Давай так назовём, это уже третий раз, когда я остановился и начал думать что это за модель и второй раз когда хочу написать об этом, кажется, в следующий раз должно быть проще понять.

Хотя по-хорошему, нужно импортировать просто модуль и использовать его так: `telegram.User`, `telegram.Message`, тогда вообще никогда не будет проблем с пониманием

Reply:
Изменил импорты